mergebom_class.py

```diff
@@ -357,30 +358,47 @@ def statistics(self):
     def merge(self):
         self.group()
         self.count()
+        d = {}
+        listApp = []
```

Comment:
Queste variabili vendono utilizzate fuori dal for? se no, perchè le dichiari qui e poi le usi solo all'interno dei cicli?